learn/what_is_meilisearch/language.md

```diff
@@ -1,42 +1,43 @@
 
-If you'd like to learn more about how different languages are processed in Meilisearch, see our [tokenizer documentation](/learn/advanced/tokenization.md).
+[Read more about our tokenizer](/learn/advanced/tokenization.md)
 
 ## Improving our language support
 
-While we have employees from all over the world at Meilisearch, we don't speak every language. In fact, we rely almost entirely on feedback from external contributors to know how our engine is performing across different languages.
+While we have employees from all over the world at Meilisearch, we don't speak every language. We rely almost entirely on feedback from external contributors to understand how our engine is performing across different languages.
 
-If you'd like to help us create a more global Meilisearch, please consider sharing your tests, results, and general feedback with us through [GitHub issues](https://github.com/meilisearch/Meilisearch/issues). Here are some of the languages that have been requested by users and their corresponding issue:
+If you'd like to request optimized support for a language that we don't currently support, please [open a discussion in our product repository](https://github.com/meilisearch/product/discussions).
 
-- [Arabic](https://github.com/meilisearch/meilisearch/issues/554)
-- [Lao](https://github.com/meilisearch/meilisearch/issues/563)
-- [Persian/Farsi](https://github.com/meilisearch/meilisearch/issues/553)
-- [Thai](https://github.com/meilisearch/meilisearch/issues/864)
-
-If you'd like us to add or improve support for a language that isn't in the above list, please create an [issue](https://github.com/meilisearch/meilisearch/issues/new?assignees=&labels=&template=feature_request.md&title=) saying so, and then make a [pull request on the documentation](https://github.com/meilisearch/documentation/edit/master/reference/features/language.md) to add it to the above list.
+If you'd like to help by developing a tokenizer pipeline yourself: first of all, thank you! We recommend that you take a look at the [tokenizer contribution guide](https://github.com/meilisearch/charabia/blob/main/CONTRIBUTING.md) before making a PR.
 
 ## FAQ
 
 ### What do you mean when you say Meilisearch offers _optimized_ support for a language?
 
-Under the hood, Meilisearch relies on tokenizers that identify the most important parts of each document in a given dataset. We currently use two tokenization pipelines: one for languages that separate words with spaces and one specifically tailored for Chinese. Languages that delimit their words in other ways will still work, but the quality and relevancy of search results may vary significantly.
+Under the hood, Meilisearch relies on tokenizers that identify the most important parts of each document in a given dataset. We currently use four tokenization pipelines:
+
+- A default one designed for languages that separate words with spaces
```

Comment:
Pet peeve: I really dislike most occurrences of this pattern: "A/The [adjective] one". I find it so clunky and vague.

Ergo, I'm making a super important and 100% objective suggestion:

```suggestion
- A default pipeline designed for languages that separate words with spaces
```